knative-operator/pkg/controller/knativekafka/knativekafka_controller_test.go

```diff
@@ -152,11 +94,23 @@ func TestKnativeKafkaReconcile(t *testing.T) {
 			initObjs := []runtime.Object{test.instance}
 
 			cl := fake.NewFakeClient(initObjs...)
-			mgr := &fakeManager{
-				client: cl,
-				scheme: s,
+
+			kafkaChannelManifest, err := mf.ManifestFrom(mf.Path(os.Getenv("KAFKACHANNEL_MANIFEST_PATH")))
```

Comment:
With this in place, you can also drop the os.Setenv commands on top for good measure, I believe.

Reply:
:facepalm: 
that's copy paste w/o thinking :facepalm: